hail/python/hail/context.py

```diff
@@ -4,7 +4,7 @@
 import hail
 from hail.genetics.reference_genome import ReferenceGenome
 from hail.typecheck import nullable, typecheck, typecheck_method, enumeration
-from hail.utils import wrap_to_list, get_env_or_default
+from hail.utils import wrap_to_list, get_env_or_default, local_path_uri
```

Comment:
unused import?

Reply:
ah, yeah. Originally I set out to make the logging work naturally with hadoop files, but that's a much larger project.